easybuild/easyconfigs/s/STAR/STAR-2.3.0e-goolf-1.4.10.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'STAR'
+version = '2.3.0e' 
+
+homepage = 'https://code.google.com/p/rna-star/'
+description = """ STAR aligns RNA-seq reads to a reference genome using 
+ uncompressed suffix arrays. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://rna-star.googlecode.com/files/']
+sources = ['%(name)s_%(version)s.tgz']
+
+patches = ['STAR_missing_includes.patch']
+
+# a binary is provided with the source files. We delete it
+premakeopts = 'rm STAR && '
+
+parallel = 1
+
+files_to_copy = [(["STAR"], "bin"), "README.txt"]
+        
+sanity_check_paths = {
+    'files': ["bin/STAR"],
+    'dirs': [""],
+}
+
+moduleclass = 'bio'
+
```

Comment:
trip last empty line and prefer `'dirs': [],`
@boegel: otherwise ready to merge